src/api/organisationUnits.js

```diff
@@ -1,64 +1,115 @@
 import { onError } from './index'
 
-export const apiFetchOrganisationUnitRoot = d2 => {
-    const endPoint = '/organisationUnits'
-    const fields = ['id', 'displayName', 'name']
-    const url = `${endPoint}?paging=false&userDataViewFallback=true&fields=${fields.join(
-        ','
-    )}`
-
-    return d2.Api.getApi()
-        .get(url)
-        .then(({ organisationUnits }) => organisationUnits[0])
-        .catch(onError)
+const orgUnitLevelsQuery = {
+    resource: 'organisationUnitLevels',
+    params: ({ displayNameProp = 'displayName' }) => ({
```

Comment:
It could be worthwhile to also default to an empty object so the fn can be called without any args.
```suggestion
    params: ({ displayNameProp = 'displayName' } = {}) => ({
```